lib/airbrake/rails/active_job.rb

```diff
@@ -12,19 +12,19 @@ module ActiveJob
         end
 
         rescue_from(Exception) do |exception|
-          next unless (notice = Airbrake.build_notice(exception))
+          if (notice = Airbrake.build_notice(exception))
+            notice[:context][:component] = 'active_job'
+            notice[:context][:action] = self.class.name
 
-          notice[:context][:component] = 'active_job'
-          notice[:context][:action] = self.class.name
+            notice[:params] = as_json
 
-          notice[:params] = as_json
-
-          # We special case Resque because it kills our workers by forking, so
-          # we use synchronous delivery instead.
-          if is_resque_adapter
-            Airbrake.notify_sync(notice)
-          else
-            Airbrake.notify(notice)
+            # We special case Resque because it kills our workers by forking, so
+            # we use synchronous delivery instead.
+            if is_resque_adapter
+              Airbrake.notify_sync(notice)
+            else
+              Airbrake.notify(notice)
+            end
           end
 
           raise exception
```

Comment:
So the problem is that now we re-raise this and nothing catches it. That's why Airbrake tries to notify it one more time. `Airbrake.notify{_sync}` internally uses `build_notice`. Hope this makes sense. It would be nice to fix tests, so I could release the library tomorrow. If not, then no worries, I'll fix it myself.